src/EditorFeatures/Test2/Rename/RenameCommandHandlerTests.vb

```diff
@@ -1033,5 +1033,463 @@ partial class [|Program|]
                 Assert.Null(workspace.GetService(Of IInlineRenameService).ActiveSession)
             End Using
         End Sub
+
```

Comment:
Can we not add 458 lines of code to this file? Perhaps a refactoring is in order?